packages/relay/src/lib/eth.ts

```diff
@@ -86,6 +87,7 @@ export class EthImpl implements Eth {
   static ethFeeHistory = 'eth_feeHistory';
   static ethGetTransactionCount = 'eth_getTransactionCount';
   static ethSendRawTransaction = 'eth_sendRawTransaction';
+  static executionReverted = 'execution reverted: ';
```

Comment:
If you need this add it in the JSONRPCError class and call from here so the string is centralized?

Reply:
Removed